### PR TITLE
WFNEWS-2277 : Use redirect for links to event page and formatting

### PR DESCRIPTION
**Pull request by sukhpalp**

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/preview-panels/danger-rating-preview/danger-rating-preview.component.ts [hunk 1]
```diff
@@ -44,44 +48,43 @@ export class DangerRatingPreviewComponent {
...
+          }
+        }
+      });
   }
 
 
   displayDangerRatingDes(danger) {
     switch (danger) {
       case 'Extreme':
+        // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/preview-panels/danger-rating-preview/danger-rating-preview.component.ts:80]: This line can be removed

File: client/wfnews-war/src/main/angular/src/app/components/preview-panels/danger-rating-preview/danger-rating-preview.component.ts [hunk 2]
```diff
@@ -44,44 +48,43 @@ export class DangerRatingPreviewComponent {
...
         return 'Extremely high risk of fire starting. Forest fuels are extremely dry and the fire risk is very serious. Fires can start easily, spread rapidly, and challenge fire suppression efforts. Forest activities may be restricted.';
       case 'High':
         return 'Serious risk of fire starting. Forest fuels are very dry and extreme caution must be used.';
       case 'Moderate':
         return 'Moderate risk of fire starting. Forest fuels are dry and caution should be exercised in forested areas.';
       case 'Low':
+        // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/preview-panels/danger-rating-preview/danger-rating-preview.component.ts:87]: And this line

File: client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.stories.ts [hunk 3]
```diff
@@ -1,75 +1,76 @@
...
+    fireCentreName: 'Coastal Fire Centre',
+    fireOfNoteInd: true,
+    wasFireOfNoteInd: true,
+    incidentName: 'NOTIFICATION SHARON',
+    incidentLocation: 'Near Juan De Fuca Park',
+    traditionalTerritoryDetail: null,
+    incidentSizeEstimatedHa: 43.222,
+    incidentSizeMappedHa: 43.222,
+    incidentSizeDetail: 'Fire size is based on most current information available. 232',
+    // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.stories.ts:29]: This also

File: client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.stories.ts [hunk 4]
```diff
@@ -1,75 +1,76 @@
...
+    lastUpdatedTimestamp: 1715793217397,
+    createDate: 1682630478374,
+    updateDate: 1715793217402,
+    latitude: '48.529335',
+    longitude: '-124.448432',
+    fireYear: 2023,
+    responseTypeCode: null,
+    responseTypeDetail: null,
+    fireZoneUnitIdentifier: 54,
+    // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-info-panel/incident-info-panel.stories.ts:51]: And here

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list-header/bans-list/bans-list.component.ts [hunk 5]
```diff
@@ -80,6 +80,7 @@ export class BansListComponent implements OnInit {
     let whereString = '';
 
     if (this.searchText && this.searchText.length > 0) {
+      // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list-header/bans-list/bans-list.component.ts:83]: Here also

File: client/wfnews-war/src/main/angular/src/app/components/wildfires-list-header/evac-list/evac-list.component.ts [hunk 6]
```diff
@@ -90,6 +90,7 @@ export class EvacListComponent implements OnInit {
     let whereString = '';
 
     if (this.searchText && this.searchText.length > 0) {
+      // eslint-disable-next-line max-len
```
Comment (ssylver93) [inline client/wfnews-war/src/main/angular/src/app/components/wildfires-list-header/evac-list/evac-list.component.ts:93]: Here also

**Review by ssylver93 — APPROVED** [review]
Looks good, just a few spots where commented eslint lines have been left in

**Comment by sukhpalp** [comment]
These comments serve a purpose. They tell the linter, that I acknowledge that the following line is breaking the rule. That way it will not be marked as an issue. In most of these cases, its a long string being used for copy or in a story as an example. This will reduce issues in SonarCloud.

**Pull request merged**